frontend/packages/integration-tests-cypress/cy-command-additions/login.ts

```diff
@@ -0,0 +1,51 @@
+import { submitButton } from '../utils/selectors';
+
+declare global {
+  // eslint-disable-next-line @typescript-eslint/no-namespace,no-redeclare
+  namespace Cypress {
+    interface Chainable<Subject> {
+      login(providerName?: string, username?: string, password?: string): Chainable<Element>;
+      logout(): Chainable<Element>;
+    }
+  }
+}
+
+const usernameInput = '#inputUsername';
+const passwordInput = '#inputPassword';
+const KUBEADMIN_IDP = 'kube:admin';
```

Comment:
Let's read this from an env var `BRIDGE_KUBEADMIN_IDP` and set it in the test prow script.

Reply:
fixed